TestApp/Source/Android/app/src/main/cpp/BabylonNativeJNI.cpp

```diff
@@ -61,30 +84,20 @@ Java_com_android_appviewer_AndroidViewAppActivity_finishEngine(JNIEnv* env, jobj
 {
 }
 
-void LogMessage(const char* message, Babylon::LogLevel level)
-{
-    switch (level)
-    {
-    case Babylon::LogLevel::Log:
-        __android_log_write(ANDROID_LOG_INFO, "BabylonNative", message);
-        break;
-    case Babylon::LogLevel::Warn:
-        __android_log_write(ANDROID_LOG_WARN, "BabylonNative", message);
-        break;
-    case Babylon::LogLevel::Error:
-        __android_log_write(ANDROID_LOG_ERROR, "BabylonNative", message);
-        break;
-    }
-}
-
 JNIEXPORT void JNICALL
 Java_com_android_appviewer_AndroidViewAppActivity_surfaceCreated(JNIEnv* env, jobject obj, jobject surface)
 {
     if (!runtime)
     {
         ANativeWindow *window = ANativeWindow_fromSurface(env, surface);
 
-        runtime = std::make_unique<Babylon::RuntimeAndroid>(window, "file:///data/local/tmp", LogMessage);
+        runtime = std::make_unique<Babylon::RuntimeAndroid>(window, "file:///data/local/tmp");
+
+        runtime->Dispatch([](Babylon::Env& env)
```

Comment:
I'm wondering if it's possible to have a simpler function like 'AddPlugin' that would be called like this:
```
runtime->AddPlugin("Console", logHandler);
```
And do the internal cooking with env.Global in Runtime.

Reply:
I was thinking it's more like a helper static function instead of an instance function:

```
    AddConsole(runtime, logHandler);
```

Maybe under a namespace also.